NuGet.Config

```diff
@@ -3,4 +3,7 @@
   <config>
     <add key="repositoryPath" value="packages" />
   </config>
+  <packageSources>
+    <add key="msreact-public" value="https://pkgs.dev.azure.com/ms/react-native/_packaging/react-native-public/nuget/v3/index.json" />
```

Comment:
Why is there something related to ReactNative here? This should not depend on React Native.

Reply:
Yeah - I am not sure what it is required as well.

Reply:
@adamkrantz 

That's just the publicly available feed for packages **used by** React Native for Windows that we can't/decided not to publish in nuget.org.

It does not introduce a dependency on React Native.

Reply:
> I'm concerned about having this depend on ReactNative Windows for ARM64

@akrantz This change doesn't introduce any dependencies on React Native for Windows, or React Native itself.
The package name only indicates "who produced it", or "what project it's for".

Is that enough context to unblock this PR?